src/Microsoft.DotNet.Build.Tasks/PackageFiles/PackageLibs.targets

```diff
@@ -60,6 +60,12 @@
     <Error Condition="'$(IsReferenceAssembly)' == 'true' AND '$(PackageTargetRuntime)' != ''"
            Text="Reference assemblies should not specify the PackageTargetRuntime property since runtimes cannot effect compile time assets."/>
 
+    <!-- PackageTargetRuntime Defaults -->
+    <PropertyGroup Condition="'$(UsePackageTargetRuntimeDefaults)' == 'true' AND '$(PackageTargetRuntime)' == ''">
+      <PackageTargetRuntime Condition=" '$(TargetsWindows)' == 'true'">win7</PackageTargetRuntime>
```

Comment:
nit: there's an additional space in front of this condition that isn't there in the next line